mlflow/types/schema.py

```diff
@@ -52,6 +53,8 @@ def __new__(cls, value, numpy_type, spark_type, pandas_type=None, python_type=No
         dt.date,
     )
     """64b datetime data."""
+    pilimage = (9, "PILImage", "PILImage", object, Image.Image)
+    """pil image."""
```

Comment:
Please revert this change, we shouldn't expect image to be one of the DataTypes, also we should not infer it directly as a type. You should pass string representing the path to the image, or the bytes, instead of the image itself.

Reply:
@serena-ruan  transformers imagepipelines accept PIL Image also as input. Wondering if we should also support it in mlflow models??

Reply:
I don't think we should support it in signatures. For general input, I feel like most cases users just pass in the path to the image, and you can always use bytes. If there're asks for this image type, we can discuss later :)